### PR TITLE
FRONTEND - Vi/frontend navigation

### LINKED ISSUE #95: Create navigation html/css/vue + empty tabs for content

### PR DESCRIPTION
**Pull request by vincentvis**
First frontend PRRRRRRRR

some cleanup
some default css stuff

navigation bar #95 
and empty tabs! (well.. with a title..)

closes #95

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -1,18 +1,19 @@
 services:
     frontend:
-        container_name: "frontend_${COMPOSE_PROJECT_NAME:-ts}"
+        container_name: 'frontend_${COMPOSE_PROJECT_NAME:-ts}'
```
Comment (Ryno95) [inline compose.yml:3]: Appending the -ts, does that solve compiling across multiple versions of the project? I think you explained it before but not specifically what changed

File: frontend/src/components/__tests__/not-a-test.spec.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { describe, it, expect } from 'vitest';
+
+describe('Not a Test', () => {
+	it('is not testing anything', () => {
+		expect(true).toBeTruthy();
```
Comment (Ryno95) [inline frontend/src/components/__tests__/not-a-test.spec.ts:5]: 🤣

**Review by Ryno95 — APPROVED** [review]
Curiousity question... Why do we have 7 of the pong images? I could only find two of them being used in `frontend/public/site.webmanifest`

Thanks for setting up the template for the frontend, feels like 80% of the project is now done 🤓😬🤫

File: compose.yml (hunk 1 above)
Comment (vincentvis) [reply compose.yml:3]: this adds a `-ts` suffix when the COMPOSE_PROJECT_NAME is not set (which it probably isnt.)
so if you run compose like this:
`COMPOSE_PROJECT_NAME=CHECKPR docker compose up --build`
it appends `CHECKPR` to all the containers to stop it from complaining about "container already in use"

**Comment by vincentvis** [comment]
> Curiousity question... Why do we have 7 of the pong images? I could only find two of them being used in `frontend/public/site.webmanifest`
> 
> Thanks for setting up the template for the frontend, feels like 80% of the project is now done 🤓😬🤫

Most of those images are automagically selected by the browser/device that wants them
This is all generated by [realfavicongenerator](realfavicongenerator.net/) and I just copy paste the files etc from there :)

File: frontend/src/assets/main.css [hunk 3]
```diff
@@ -1,35 +1,59 @@
 @import './base.css';
```
Comment (livlaming) [inline frontend/src/assets/main.css:2]: not sure where to place this comment, the menu looks amazing!
If we will be using the lined square maybe its nice to make is a bit more in the same ratio as the design? (a bit more squarisch but not fully)

Comment (vincentvis) [reply frontend/src/assets/main.css:2]: which lined square? I did nothing with a lines square 🙃

File: frontend/public/browserconfig.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (vincentvis) [reply frontend/public/browserconfig.xml:1]: it's just an icon for when you add it to your homescreen on an ios device

[the remark this answers is not in the document]
Comment (livlaming) [reply frontend/public/browserconfig.xml:1]: haha alright fun!

File: frontend/src/assets/main.css (hunk 3 above)
Comment (livlaming) [reply frontend/src/assets/main.css:2]: Screen Shot 2023-01-30 at 3 26 47 PM

Comment (livlaming) [reply frontend/src/assets/main.css:2]: deze rechthoek rondom this is a login page

Comment (vincentvis) [reply frontend/src/assets/main.css:2]: I guess that is indeed part of this first setup thingy :P 
I honestly just made a box, not really worrying about the size, but I adjusted it now to be 4/5 aspect ratio (which is close to what it is in penpot)

Comment (livlaming) [reply frontend/src/assets/main.css:2]: perfect!

**Pull request merged**